parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -293,7 +293,16 @@ ensureDocker() {
         wait_for_file 1200 1 $DOCKER_MOUNT_FLAGS_SYSTEMD_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     fi
     DOCKER_JSON_FILE=/etc/docker/daemon.json
-    wait_for_file 1200 1 $DOCKER_JSON_FILE || exit $ERR_FILE_WATCH_TIMEOUT
+    for i in $(seq 1 1200); do
```

Comment:
Because this is JSON and doesn't support line comments, we have to manually validate this (instead of being able to use the `wait_for_file` convenience.